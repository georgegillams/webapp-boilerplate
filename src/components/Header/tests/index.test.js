import React from 'react';
import { render } from '@testing-library/react';

import { Header } from '..';

describe('<Header />', () => {
  it('Should render correctly', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
