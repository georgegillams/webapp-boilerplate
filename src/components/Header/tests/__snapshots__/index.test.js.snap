// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render correctly 1`] = `
<div>
  <header
    class="css-w41d0-HeaderRoot ez4dfsd0"
  >
    <div
      class="css-u5y1t0-HeaderContainer ez4dfsd1"
    >
      <nav
        class="css-wvyf6s-NavRoot ez4dfsd2"
      >
        <a
          href="/"
        >
          Home
        </a>
        <div
          class="css-kh7q0g-Space ez4dfsd4"
        />
        <a
          href="/account"
        >
          Account
        </a>
        <div
          class="css-kh7q0g-Space ez4dfsd4"
        />
        <div
          class="css-1jbsa08-ListActions ez4dfsd3"
        />
      </nav>
    </div>
  </header>
</div>
`;
